File: 100-Days-Series/Day 31/ToggleCase.py
#Function For ToggleCase
def getToggleCase(str):
    result=""
    for i in range (0,len(str),1):
        ch=str[i]
        if ch.isupper():
            result+=ch.lower()
        elif ch.islower():
            result+=ch.upper()
        else:
            result+=ch
    return result
#Let's Start here
str="Hello, World!"
print("Original String : ",str)
toggledString=getToggleCase(str)
print("Toggled String : ",toggledString)

File: 100-Days-Series/Day 31/ToggleCase.c
#include<stdio.h>
#include<ctype.h>
#include<string.h>
// Method for ToggleCase
void getToggleCase(char *str){
    while(*str){
        if(isupper(*str)){
            *str=tolower(*str);
        }else if(islower(*str)){
            *str=toupper(*str);
        }
        str++;// Move to the next character
    }
}
// Let's Start here
void main(){
    char str[]="Hello, World!";
    printf("\nOriginal String : %s",str);
    getToggleCase(str);
    printf("\nToggled String : %s",str);

}